tests/test_cells.py

```diff
@@ -40,3 +41,21 @@ def test_set_cell_size_infinite():
             )
             == size
         )
+
+
+def test_chop_cells():
+    """Simple example of splitting cells into lines of width 3."""
+    text = "abcdefghijk"
+    assert chop_cells(text, 3) == ["abc", "def", "ghi", "jk"]
+
+
+def test_chop_cells_double_width_boundary():
+    """The available width lies within a double-width character."""
+    text = "ありがとう"
+    assert chop_cells(text, 3) == ["あ", "り", "が", "と", "う"]
+
+
+def test_chop_cells_mixed_width():
+    """Mixed single and double-width characters."""
+    text = "あ1り2が3と4う56"
+    assert chop_cells(text, 3) == ["あ1", "り2", "が3", "と4", "う5", "6"]
```

Comment:
I mixed it up a bit more to be less regular.
(I did this by hand and didn't run the test after! I also just copied and pasted some characters, maybe there was some meaning in here that I butchered... 😬)

```suggestion
    text = "あ1り2がり34とりう56"
    assert chop_cells(text, 3) == ["あ1", "り2", "が", "り3", "4と", "り", "う5", "6"]
```

Reply:
You turned "arigatou" into "arigaritoriu" 😄